### ISSUE TITLE
Avatars

### ISSUE THREAD

**Issue by paulsonder** [OpenConceptLab/ocl_issues#1905]
Document use of avatars, including:

- [ ] Orgs that have not uploaded an avatar
- [ ] Users that have not uploaded an avatar
- [ ] How to update, including cropping
- [ ] Specs and dimensions for avatars across the application

**Comment by paulsonder** [comment]
@snyaggarwal I created a ticket for this and it could be fun to work on it together - I know you had some ideas early on.

I'm thinking this can cover avatar specs, including uploading your own, but also maybe a library of avatars for users to choose from.

**Comment by snyaggarwal** [comment]
Yep something like this - https://getavataaars.com/

**Comment by paulsonder** [comment]
Yeah! I think the user generated content quality can be a little low sometimes, so if we can provide a range of assets in a range of colors and terminology related, it could improve the visual aesthetic of OCL Online overall.

**Comment by paulsonder** [comment]
![image](https://github.com/user-attachments/assets/5abd38e9-8e1c-4850-84ed-316e9c571d50)

**Comment by jamlung-ri** [comment]
Love that! I would be very supportive of using that.

**Comment by paulsonder** [comment]
We can capture ideas and requirements here for post-MVP avatar work
